mongo/integration/initial_dns_seedlist_discovery_test.go

```diff
@@ -62,13 +62,13 @@ func runSeedlistDiscoveryTest(mt *mtest.T, file string) {
 		mt.Skip("skipping to avoid go1.11 problem with multiple strings in one TXT record")
 	}
 
-	cs, err := connstring.Parse(test.URI)
+	cs, err := connstring.ParseAndValidate(test.URI)
```

Comment:
This was changed to `Parse` in a previous commit because there was a URI that contained an auth source without a username, which caused `ParseAndValidate` to report an error, but the test did not expect an error. That was resolved by a recent spec change to allow this combination and using `ParseAndValidate` is necessary because one of the new seedlist discovery tests added in this change asserts that a validation error occurred.

Reply:
Ack